.github/workflows/main.yml

```diff
@@ -98,6 +98,19 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
+    - name: Cache datasets
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-datasets
+      with:
+        path: |
+          data
+          ~/tensorflow_datasets
+        key: build-${{ env.cache-name }}
+        restore-keys: |
+          build-${{ env.cache-name }}
```

Comment:
We should add the same cache in `test-benchmark`

Reply:
yes, done